executor/runtime/docker/lxcfs.go

```diff
@@ -13,11 +13,19 @@ const (
 
 func getLXCFsBindMounts() []string {
 	extraBinds := []string{}
-	for _, file := range []string{"cpuinfo", "meminfo", "uptime"} {
-		path := filepath.Join(lxcfs, "proc", file)
+	lxcfsEndpoints := []string{
```

Comment:
I was looking at info about lxcfs and noticed [this page](https://github.com/lxc/lxcfs#introduction) which sounds like lxcfs tries to generate container-scoped stats. Are these bind mounts in support of that, or are these instead for "punching a hole" up to the host-level stats? FWIW that page also lists `/sys/devices/system/cpu` (no idea how this interacts with `/sys/devices/system/cpu/online`) and `/proc/slabinfo`.

PS: could alphabetize the list - but the above lxcfs doc doesn't fully alphabetize theirs either

Reply:
> Are these bind mounts in support of that,

Yes.

> /sys/devices/system/cpu

We can't easily take a version that supports those files, due to the libfuse3 dependency.

Reply:
Finally getting back to this:

> PS: could alphabetize the list - but the above lxcfs doc doesn't fully alphabetize theirs either

I went ahead and put them in the order that the lxcfs readme has them, so that hopefully it's easier to compare when someone in the future is curious.